.gitignore

```diff
@@ -1,3 +1,4 @@
+xcframeworks
```

Comment:
nit: put under one of the headings, maybe **Build assets**

Reply:
I tried to remove this commit. It should not be in this branch. I will try again. GitHub is not allowing to to modify the branch and force push like I've done before. Normally I can clean up a branch.